Code/Tools/SceneAPI/SceneCore/Containers/Scene.h

```diff
@@ -33,7 +33,7 @@ namespace AZ
                 void SetSource(const AZStd::string& filename, const Uuid& guid);
                 void SetSource(AZStd::string&& filename, const Uuid& guid);
                 const AZStd::string& GetSourceFilename() const;
-                const AZStd::string GetSourceExtension() const;
+                const AZStd::string_view GetSourceExtension() const;
```

Comment:
The const on the return value is unnecessary here and is ignored, because the rreturn type is a value type.
So users can still do `AZStd::string_view ext = scene->GetSourceExtension()`

Reply:
Agh, thank you, change made before consuming coffee this morning. :( Fixed!